### PR TITLE
Replace deprecated directories crate with directories-next

### LINKED ISSUE #3298: RUSTSEC-2020-0054: directories is unmaintained, use directoriess-next instead
> directories is unmaintained, use directoriess-next instead

| Details             |                                                |
| ------------------- | ---------------------------------------------- |
| Status              | unmaintained                |
| Package             | `directories`                      |
| Version             | `2.0.2`                   |
| URL                 | [https://github.com/dirs-dev/directories-rs](https://github.com/dirs-dev/directories-rs) |
| Date                | 2020-10-16                         |

The `directories` crate is not maintained any more;
use [`directories-next`](https://crates.io/crates/directories-next) instead.

See [advisory page](https://rustsec.org/advisories/RUSTSEC-2020-0054.html) for additional details.

### LINKED ISSUE #3299: RUSTSEC-2020-0054: directories is unmaintained, use directoriess-next instead
> directories is unmaintained, use directoriess-next instead

| Details             |                                                |
| ------------------- | ---------------------------------------------- |
| Status              | unmaintained                |
| Package             | `directories`                      |
| Version             | `3.0.1`                   |
| URL                 | [https://github.com/dirs-dev/directories-rs](https://github.com/dirs-dev/directories-rs) |
| Date                | 2020-10-16                         |

The `directories` crate is not maintained any more;
use [`directories-next`](https://crates.io/crates/directories-next) instead.

See [advisory page](https://rustsec.org/advisories/RUSTSEC-2020-0054.html) for additional details.

### PR DESCRIPTION
**Pull request by D4nte**
See https://rustsec.org/advisories/RUSTSEC-2020-0054.html

Resolves #3298
Resolves #3299

### CODE REVIEW

File: cnd/Cargo.toml [hunk 1]
```diff
@@ -19,7 +19,7 @@ derivative = "2"
 diesel = { version = "1.4", features = ["sqlite"] }
 diesel_migrations = "1.4.0"
 digest = { path = "../digest" }
-directories = "3.0"
+directories-next = "2"
```
Comment (tcharding) [inline cnd/Cargo.toml:22]:
```suggestion
directories-next = "2.0"
```

**Review by tcharding — APPROVED** [review]
Nice one, I think we should include the MINOR version number `2.0`.

**Comment by D4nte** [comment]
> Nice one, I think we should include the MINOR version number `2.0`.

Why? 🤔

**Comment by tcharding** [comment]
> > Nice one, I think we should include the MINOR version number `2.0`.
> 
> Why?

Oh, turns out I'm wrong. Minor version is supposed to be only backward compatible changes so you are right, we don't need to be explicit on this.

Thanks

**Pull request merged**